CondFormats/SiPixelObjects/interface/SiPixelGainForHLTonGPU.h

```diff
@@ -46,7 +47,7 @@ class SiPixelGainForHLTonGPU {
     assert(offset < 3088384);
```

Comment:
this is awful, please expand into 3088384 = 1856*416*4
Also explain what each piece stands for.

Reply:
I have no idea what each piece stands for ...